pegasus/sites.v3/code.org/views/learn_carousel_block.haml

```diff
@@ -5,7 +5,7 @@
     - launch_path = "/api/hour/begin/starwarsblocks"
   - else
     - launch_path = "/starwars"
-- if id == "gumball" || id == "iceage" || id == "playlab"
+- elsif id == "gumball" || id == "iceage" || id == "playlab"
```

Comment:
incidentally do the gumball & iceage paths get exercised at this point, or just playlab?